modules/UI/toolbars/Toolbar.js

```diff
@@ -2,6 +2,7 @@
 /* jshint -W101 */
 import UIUtil from '../util/UIUtil';
 import UIEvents from '../../../service/UI/UIEvents';
+import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
```

Comment:
Since KeyboardShortcut is separate module isn't it better to be attached to the global APP object like other modules. That way you should access it with `APP. KeyboardShortcut` instead of `import`. WDYT?